srcpkgs/ibus-bamboo/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'ibus-bamboo'
+pkgname=ibus-bamboo
+version=0.4.6
+revision=1
+archs=noarch
```

Comment:
This has to be wrong, `noarch` is for platform independent stuff, golang is a compiled language and the binaries are platform dependent.

Reply:
So it should be `arch=*` then?

Reply:
The default is all architectures, so just remove it.